submissions/no pain no gain/no pain no gain.md

```diff
@@ -0,0 +1,78 @@
+
+```mermaid !
+pie title NETFLIX
+         "Cliffs" : 382
+         "Mixed belts (by mistake)" : 284
+         "Bots (recharging speed, speed, buildings limits, overall)" : 134
+         "Trees and removing them" : 168
+         "The entire fluid system" : 225
+         "Power poles spaghetti connecting" : 397
+```
+
+Thanks to every 1.6k people who took part in the survey and expressed their opinion. Results shows that the most annoying things for players is "Power poles spaghetti connecting". I don't think anyone was surprised that power poles will be one of the most annoying thing, because everyone try to keep their factory in perfect by using good ratios, proper trains names or by building a "main bus" design.
+
+![](https://i.imgur.com/Ey0mvSp.png)
+
+Blueprints however don't save connected cables and no matter how pretty they look when you creating a blueprint, this one little detail will not be saved. It's getting more noticeable, when you're entering megabase stadium and most of your buildings are made out of blueprints.
+
+![](https://i.imgur.com/I0HQ0RS.png)
+
+We have to look also at most upvoted comments, because they might have less votes then answers from pool, but less people look into comments section. Putting first one aside, second one is a proper submission with big amount of upvotes.
+
+![](https://i.imgur.com/HHWMOlb.png)
+
+Power armors give player additional 10-30 inventory size bonus depending on tier. This space is very useful to keep more machines and build bigger buildings, but keeping whole inventory full sometimes leads to unpleased situations. There is not much to say because [u/Mr_Shteeveey](https://www.reddit.com/user/Mr_Shteeveey/) described it very well in his comment.
+
+![](https://i.imgur.com/x8jG8go.png)
+
+### Painful things that got removed
+
+Every game developer tries to make their game the best on the market. Factorio devs put a lot of effort to make their game very intuitive and fun to play without taking advanced mechanics from the game. I think many people would agree that factorio is "easy to play, hard to master" type of game, because you can start playing it without any knowledge and know what you are doing and what you need to do next, but it wasn't always like that.
+
+Factorio had a lot of annoying mechanics, that every player hated. I don't know if many of you remember killing biter bases to get some [Alien artifact](https://wiki.factorio.com/Alien_artifact), because that feature was deleted in 0.15 (24.04.2017). For those who don't remember, alien artifacts was used to craft advance materials like modules tier 3 or power armors and also you could craft [purple science](https://wiki.factorio.com/Alien_science_pack), that was the highest tier of science packs in Factorio version 0.15. Constant expansion for more purple orbs wasn't in "Factorio style" where idea is to automate everything, so they got removed and very well.
```

Comment:
Possibly reference the nauvis archives about purple goop from a previous issue here?